apps/backend/src/app/controllers/run/run.resolver.ts

```diff
@@ -1,8 +1,32 @@
-import { Resolver } from "@nestjs/graphql";
-// import { Run as GQLRun } from "../../types/graphql";
+import { Resolver, Mutation, Args } from "@nestjs/graphql";
 import { RunService } from "./run.service";
 
 @Resolver("Run")
 export class RunResolver {
   constructor(private readonly runService: RunService) {}
+
+  @Mutation()
+  async createRun(
+    @Args("userId") userId: string,
+    @Args("maxHR") maxHR: number,
+    @Args("token") token: string,
```

Comment:
These things should be extracted from the @User(), not as arguments to the mutation.